collector/docker/kindling-collector-config.yml

```
@@ -228,6 +228,13 @@ exporters:
       endpoint: 10.10.10.10:8080
     stdout:
       collect_period: 15s
+    memcleanup:
+      # unit: Hour
+      # set 0 to disable
+      restart_period: 12
+      # unit: Day
+      # set 0 to disable (restart at night)
+      restart_every_n_days: 0
```

Please add a separate option for controlling whether to enable this feature.
```suggestion
    # This feature will reset the otelexporter periodicity to clear the metrics saved in memory.
    memcleanup:
      # Whether to enable this feature. False by default.
      enable: false
      # unit: Hour
      restart_period: 12
      # unit: Day
      # set 0 to disable (restart at night)
      restart_every_n_days: 0
```